### PR TITLE
WIP: if IE => display browser support msg

### PR DESCRIPTION
**Pull request by sophieyang0813**
WIP PR for #311. I plan to use the online IE emulator to test this code. 

Browser icons(fontawesome) and Message content are chosen tentatively. Please let me know how I can update them.
screen shot 2018-08-21 at 1 24 11 pm

### CODE REVIEW

**Comment by sophieyang0813** [comment]
@nbiton  IE emuator shows supported browser icons and message content, while not displaying unee-T icon & bondi blue background color. I will look into how to display these. But let me know if you have any suggestion 😄😄 

screen shot 2018-08-23 at 11 23 57 am

**Comment by sophieyang0813** [comment]
@nbiton @kiatlim @michkwt Is there a single path SVG for Unee-T logo? 
 
Unee-T Logo is not showing due to missing `<g>`tag when rendered in IE. One [workaround](https://github.com/facebook/react/issues/7358) is a simpler, single-path SVG. I also come across as a solution dangerouslySetInnerHTML. 

IE 11 (Window 10): Unee-T logo not displaying  
screen shot 2018-08-27 at 12 09 28 pm
note: IE v.9 was not tested due to reaching daily free usage quota at Browserling ([IE Emulator](https://www.browserling.com/internet-explorer-testing)). Only IE v.11 (Window 10) was available at [crossBrowserTesting](https://app.crossbrowsertesting.com/).

**Comment by nbiton** [comment]
I think we should produce a simple png logo to ensure maximum support

**Comment by sophieyang0813** [comment]
@nbiton  This code is ready for review. I chose to use png instead of svg. Let me know what you think. Thank you @kiatlim for spending this afternoon looking into single path svg logo. 

IE 11 (Window 10): Current look with Unee-T logo png 
screen shot 2018-08-27 at 3 28 44 pm
[tested on crossBrowserTesting](https://app.crossbrowsertesting.com/livetests/run)

**Comment by nbiton** [comment]
The only issue I see with how this is structured, is that it only blocks the login page from showing up with this message. If somebody clicks an email invitation or gets some deep link some other way, it will still try to load regularly and look faulty to the user.
I think it should be moved to the *app.jsx* level to make sure all app routes are blocked by the message.

**Comment by sophieyang0813** [comment]
@nbiton This code is ready for review. Browser support msg is now at app.jsx level. I tested  `/signup` and `/forgot-pass` with crossBrowserTesting(IE emulator) and it works as intended 😄

File: imports/ui/app.jsx [hunk 1]
```diff
@@ -33,8 +34,12 @@ class App extends Component {
   render () {
     const { userLoggedIn, errors, dispatch } = this.props
     const firstError = errors.length ? errors[0] : ''
+    var isIE = /*
+    @cc_on!@
+    */false || !!document.documentMode
     return (
       <div className='roboto'>
+        {isIE && <BrowserSupportMsg />}
         {userLoggedIn ? (
```
Comment (nbiton) [inline imports/ui/app.jsx]: This would still be rendered below the browser support message. It doesn't need to be

**Comment by sophieyang0813** [comment]
@nbiton Evening! This code is ready for review. Is this change what you expected? when I tested, it no longer shows another page below the browser support msg.

**Comment by PicoCreator** [comment]
Took it for a spin on various IE's;  IE 10, and 11 loads alright. Except 64bit IE 10 missing the logos

As for IE 9 and below, its a lost cause. So unless you plan to support warning those (hopefully not), you can consider this closed

**Windows 7 64bit - IE 10**
Missing icons : note this only happens for 64bit edition, which frankly is a very small minority. As early stages 64 bit adopters tend to be computer enthusiast - and frankly non IE users.

windows_7_64-bit_internet_explorer_10_1280x768_full

> Here are various working IE10/11

**Windows 7 - IE 11**
windows_7_64-bit_internet_explorer_11_1280x768_full

**Windows 10 - IE 11**
windows_10_internet_explorer_11_1280x768_full

**Windows 8 - IE 10**
windows_8_internet_explorer_10_1280x768_full

**Windows 8 - IE 11**
windows_8 1_internet_explorer_11_1280x768_full

**Windows 7 - IE 11**
windows_7_64-bit_internet_explorer_11_1280x768_chromeless

**Windows 8 - IE 10**
windows_8_internet_explorer_10_1280x768_full

**Windows 8 - IE 11**
windows_8 1_internet_explorer_11_1280x768_full

> IE 9 / 8 - completely not loading in all versions

**Windows 7 64 bit - IE 9**
windows_7_64-bit_internet_explorer_9_ 64-bit _1280x768_full

**Windows 7 - IE 9**
windows_7_internet_explorer_9_1280x768_full

**Windows Vista - IE 9**
windows_vista_internet_explorer_9_1280x768_full

**Windows Vista - IE 8**
windows_vista_internet_explorer_8_1280x768_full

**Windows 7 - IE 8**
windows_7_internet_explorer_8_1280x768_full

**Comment by sophieyang0813** [comment]
@nbiton Do we plan to support IE 9/8? IE 11, 10 (except 64 bit IE 10) work as intended (Thank you, @PicoCreator for testing).

**Comment by franck-boullier** [comment]
> Do we plan to support IE 9/8?

@sophieyang0813 : no

**Pull request merged**